src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="star-line stars--0">
```

Comment:
```suggestion
    <div class="stars stars--0">
```
<img width="781" alt="Screenshot 2023-01-26 at 17 21 15" src="https://user-images.githubusercontent.com/53228259/214874984-cea61858-c8d9-404b-808f-79995744312b.png">